src/OSPSuite.Core/Serialization/SimModel/Services/SimulationExportCreator.cs

```diff
@@ -27,14 +27,14 @@ public interface ICreateExportModelVisitor
       SimulationExport CreateExportFor(IModel model, SimModelExportMode exportMode);
    }
 
-   public class CreateExportModelVisitor :
+   public class SimulationExportCreator :
```

Comment:
This class was not instantiated everytime. When exporting in //, the _idMap would be cleared etc... which would lead to very weird error. Now we always get a new instance=>no more issue